thirdparty/ofqf/qoscserver.cpp

```diff
@@ -19,128 +19,128 @@
+                                          }
+                                    if ( type == 'f' ) {
+                                          value = toFloat( tmp );
+                                          i+=4;
+                                          }
+                                    //qDebug() << " got" << value;
+
+                                    if ( args.size() > 1 )
+                                          list.append( value );
+                                    else
+                                          arguments = value;
                                     }
-                              //qDebug() << " got" << value;
-        
-                              if ( args.size() > 1 )
-                                    list.append( value );
-                              else
-                                    arguments = value;
-                              }
-        
+
                               if ( args.size() > 1 )
                                     arguments = list;
+                              }
                         }
-        		      }
-        		      qDebug() << "path seems to be" << path << "args are" << args << ":" << arguments;
-        
-              		QMap<QString,QString> replacements;
-              		replacements[ "!" ] = "^";
-              		replacements[ "{" ] = "(";
-              		replacements[ "}" ] = ")";
-              		replacements[ "," ] = "|";
-              		replacements[ "*" ] = ".*";
-              		replacements[ "?" ] = ".";
-              
-              		foreach( QString rep, replacements.keys() )
-              			path.replace( rep, replacements[ rep ] );
-              
-              		//qDebug() << " after transformation to OSC-RegExp path is" << path;
-              
-              		QRegExp exp( path );
-              		foreach( PathObject* obj, paths ) {
-              		      if ( exp.exactMatch( obj->_path ) )
-              				        obj->signalData( arguments );
-              		      }
-              		}
+                  qDebug() << "path seems to be" << path << "args are" << args << ":" << arguments;
+
+                  QMap<QString,QString> replacements;
+                  replacements[ "!" ] = "^";
+                  replacements[ "{" ] = "(";
+                  replacements[ "}" ] = ")";
+                  replacements[ "," ] = "|";
+                  replacements[ "*" ] = ".*";
+                  replacements[ "?" ] = ".";
+
+                  foreach( QString rep, replacements.keys() )
+                        path.replace( rep, replacements[ rep ] );
+
+                  //qDebug() << " after transformation to OSC-RegExp path is" << path;
+
+                  QRegExp exp( path );
+                  foreach( PathObject* obj, paths ) {
+                        if ( exp.exactMatch( obj->_path ) )
+                              obj->signalData( arguments );
+                        }
+                  }
```

Comment:
ok.  I did mention that concern in the issue report.  I guess I can just make the minimal change needed for this specific warning.